### PR TITLE
RC-v1.7.1: fix overlooked errors (type and lint) from #1738 

### PR DESCRIPTION
**Pull request by ap-justin**
Ticket(s):
-

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/pages/Registration/Steps/getRegistrationState.ts [hunk 1]
```diff
@@ -122,7 +122,7 @@ function formatDocumentation({
     isKYCRequired: KycDonorsOnly ? "Yes" : "No",
 
     //general
-    activeInCountries: ActiveInCountries,
+    activeInCountries: ActiveInCountries.map((c) => ({ label: c, value: c })),
```
Comment (ap-justin) [inline src/pages/Registration/Steps/getRegistrationState.ts:125]: `type Documentation` is already primed to be used  as `FormValue`, aws shape should be transformed here `hqCountry` , `activityCountries`

File: src/pages/Registration/Steps/Documentation/schema.ts [hunk 2]
```diff
@@ -47,5 +47,4 @@ export const schema = Yup.object().shape<SchemaShape<FormValues>>({
   hasAgreedToTerms: Yup.bool().isTrue(
     "Please confirm that you agree to our Terms and Conditions"
   ),
-  activeInCountriesOpts: Yup.array(),
```
Comment (ap-justin) [inline src/pages/Registration/Steps/Documentation/schema.ts:50]: no need see late review in #1738

**Comment by SovereignAndrey** [comment]
Thanks for the PR. Was working on this as well. :)

**Review by SovereignAndrey — APPROVED** [review]
Thanks for working on getting these last errors fixed.

**Pull request merged**